tests/unit/amlb/test_results_race_condition.py

```
@@ -0,0 +1,131 @@
+            "training_duration": 80.0,
+            "predict_duration": 5.0,
+            "models_count": 1,
+            "seed": 0,
+            "info": "",
+        }
+        board = Scoreboard(scores=[score_data], scores_dir=str(scores_dir))
+
+        # Mock file_lock to raise TimeoutError
+        from amlb.utils import process
+
+        original_file_lock = process.file_lock
```

Ruff (0.14.5)

Local variable `original_file_lock` is assigned to but never used

Remove assignment to unused variable `original_file_lock`

(F841)